modules/cudn-status-conditions.adoc

```diff
@@ -80,3 +80,157 @@ h|Message
 |`Network allocation failed for at least one node: [<node_name>], check UDN events for more info.`
 
 |===
+
+.Invalid `mtu` scenarios types for the `ClusterUserDefinedNetwork` CR
+[cols="2a,2a,3a,3a",options="header"]
+|===
+
+|Condition type
+3+|Reason, Message, Resolution
+
+.6+|`invalid mtu`
+3+|One of the following messages is returned when the `mtu` is set incorrect:
```

Comment:
`mtu` should be followed by a noun, such as "object," "resource," or "spec." The TLDR is that a specific object reference needs to be followed by a noun. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references. 

This applies throughout.

Reply:
You could also use a general reference, such as MTU instead if that is less awkward.